blog/views.py

```diff
@@ -0,0 +1,102 @@
+from django.shortcuts import render
+from blog.models import Articles ,Tags, Article_Tag_Mapping, Comments
+from django.contrib import messages
+from user.models import Users
+from django.http import HttpResponse, HttpResponseRedirect
+import random
+import string
+
+# Create your views here.
+def create(request):
+    if request.method == "POST":
+        form = (request.POST)
+        check = Users.objects.get(user_id=request.user.id)
+        slug = get_slug()
+        article = Articles(user = check, title= form["title"], body = form["body"], status = form['status'], slug = slug)
+        article.save()
+        tags = form["tags"].split(",")
+        if tags[0] != '':
```

Comment:
the condition should be len(tags) != 0. But in this case you can remove the condition